docs/adr/0122-operator-write-off-journal-storybook.md

```
@@ -0,0 +1,100 @@
+# ADR 0122: Operator Write-Off Journal Storybook
+
+**Status:** Accepted
+
+## Context
+
+#51 / ADR 0048 composes one validated write-off
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)